src/applications/financial-status-report/utils/validations.js

```diff
@@ -154,7 +154,9 @@ export const validateCurrencyArray = (errors, fieldData) => {
   }
 };
 
-export const validateIsNumber = value => {
-  const pattern = /^\d*\.?\d*$/;
-  return pattern.test(value);
+export const validateIsNumber = (errors, value) => {
+  const pattern = /^\d*$/; // This pattern ensures only whole numbers
+  if (!pattern.test(value)) {
+    errors.addError('Please enter a valid number.');
```

Comment:
This works great for when we're using the `ui:validations`, but it's causing issues on the dependent ages page because there is no `errors` to add to. I'd say leave this as is, and update the error handling on the dependent ages section